src/Components/Graphs/Graph1.jsx

```diff
@@ -1,11 +1,20 @@
 import React, { useEffect, useState, createContext } from "react";
 import axios from "axios";
 import { Column } from "@ant-design/plots";
-
+import { Oval } from 'react-loader-spinner'
+// import "react-loader-spinner/dist/loader/css/react-spinner-loader.css"
 import "./Graph1.css";
 
 function Graph1() {
   const [data, setdata] = useState([]);
+  const [loader, setLoader] = useState(false)
+
+  useEffect(() => {
```

Comment:
Fix the loader

Reply:
sameer is handeling the loader sir, i just removed my loader code